src/quilla/reports/visual_parity_report.py

```diff
@@ -75,3 +78,24 @@ def to_dict(self):
         return {
             'visualParityReport': report_data
         }
+
+    @classmethod
+    def from_dict(cls, report) -> 'VisualParityReport':
```

Comment:
Just want to confirm run id will be included as part of the report.

Reply:
Even the report is externally linked to a run id, a complete report entity should contain a run id.

Reply:
The run ID is associated with the ReportSummary, since all VP tests will be under the same run for the same file- do you think it’s better to include it in multiple places (every report contains a run ID instead of the collection of reports contains a run ID)?